### DISCUSSION TITLE
New feature: Auto-create remote groups

### DISCUSSION THREAD

**Discussion by larsks** [netbox-community/netbox#8591] [General]
Hey folks,

I submitted [this issue](https://github.com/netbox-community/netbox/issues/7671) back in October 2021 and I have a patch that implements the feature that we've been running in our Netbox instance.

I would love to contribute this feature, but I don't know the right way to get eyes on the issue.

Is this feature of interest to anyone else?

**Comment by candlerb** [comment]
Yep, it seems reasonable to me, as long as it can remained turned off (when you have many groups and don't want to create irrelevant ones in Netbox)

**Comment by larsks** [comment]
I've submitted this feature as https://github.com/netbox-community/netbox/pull/8603.  As is the case with `REMOTE_AUTH_AUTO_CREATE_USER`, this feature is turned off by default and must be explicitly enabled.

**Comment by larsks** [comment]
...which of course was rejected because the issue still isn't approved. Anyway, there it is, if people want it. I've tried to ping @jeremystretch on the issue a few times without luck; can anyone else approve it?

**Reply by candlerb** [reply]
You didn't link to the original issue in your PR. I've added the link.

**Comment by larsks** [comment]
@candlerb sorry about that, it was linked from the commit message but not the PR description. Are you able to re-open the PR?

**Reply by candlerb** [reply]
I can't, sorry.

**Reply by jeremystretch** [reply]
I've added the issue number and re-opened the PR. Please be sure to follow the PR template in the future.
